spec.bs

```diff
@@ -4639,6 +4783,29 @@ result of [=evaluating a bidding script=], or an [=additional bid=] provided by
   :: A [=duration=] in milliseconds. How long it took to run `generateBid()`.
   : <dfn>provided as additional bid</dfn>
   :: A [=boolean=], initially false.
+
+  : <dfn>bidder debug win report url</dfn>

```

Comment:
I think I asked this before, but don't see the comment or an answer.  Is there a reason not to just make these lists?  We don't treat these any differently, in this PR, at least.  Could be there are plans for different template substitutions or something, which would make this fine.

Reply:
Ack.